packages/components/src/components/calendar/views/calendar-months.jsx

```diff
@@ -5,8 +5,10 @@ import { CommonPropTypes } from './types';
 import { month_headers } from '../helpers';
 
 const Months = ({ calendar_date, isPeriodDisabled, selected_date, updateSelected }) => {
-    const moment_date = toMoment(calendar_date);
-    const moment_selected_date = toMoment(selected_date);
+    // Setting moment locale to `en` as a temporarily solution for the issue with the
+    // months names in the calendar since the month names are not localized.
```

Comment:
This is a temporary solution, Do we have any plan to refactor this date picker component? It's a very huge module which can be simplified in my opinion 🤔